andela/ah-centauri-frontend
# #164829180 Users should be able to bookmark articles

#### What does this PR do?
Allows users to bookmark articles for reading later via a bookmark button.

#### Description of Tasks to be completed?
- Users should be able to bookmark articles for reading later via a bookmark tab, button or link
- write tests for said feature

#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `feature/164829180-bookmarking-articles`
- Install dependencies:` npm install`
- Launch development server: `npm run start:dev`
- scroll to a single article. if authenticated, you will be able to see the bookmark button.

#### What are the relevant Pivotal Tracker Stories?
[#164829180](https://www.pivotaltracker.com/story/show/164829180)

#### Screenshots
###  **_Not Bookmarked_**
![image](https://user-images.githubusercontent.com/33542127/58103415-98a51f00-7beb-11e9-8770-3b731b6ed729.png)

###  **_Bookmarked_**
![image](https://user-images.githubusercontent.com/33542127/58103464-b2466680-7beb-11e9-8acb-5614fbcb2e8a.png)
